src/config.js

```diff
@@ -26,10 +26,11 @@ const env = self.AMP_CONFIG || {};
 /** @type {!Object<string, string>} */
 export const urls = {
   thirdParty: env['thirdPartyUrl'] || 'https://3p.ampproject.net',
-  thirdPartyFrameHost: env['thirdPartyFrameHost'] || 'ampproject.net',
+  thirdPartyFrameHost: env['thirdPartyFrameHost'] || 'erwinm.herokuapp.com',
   thirdPartyFrameRegex: env['thirdPartyFrameRegex'] ||
-      /^d-\d+\.ampproject\.net$/,
+      /erwinm\.herokuapp\.com/,
   cdn: env['cdnUrl'] || 'https://cdn.ampproject.org',
   errorReporting: env['errorReportingUrl'] ||
       'https://amp-error-reporting.appspot.com/r',
+  localDev: true
```

Comment:
@cramforce any other better way to figure out that its not localhost but treat the environment as `localDev` ?